app/service/file_svc.py

```diff
@@ -92,7 +92,7 @@ async def find_file_path(self, name, location=''):
                 file_path = await self.walk_file_path(os.path.join('plugins', plugin.name, subd, location), name)
                 if file_path:
                     return plugin.name, file_path
-        file_path = await self.walk_file_path(os.path.join('data'), name)
+        file_path = await self.walk_file_path(os.path.join('data', location), name)
```

Comment:
What was the impetus for this change?
(aka. uuid4 collisions in abilities vs adversaries where the data search would first return a file match in an earlier folder before defaulting to the end of the function that returns a search looking at the specific folder location)

Also, I feel like this maybe should be handled more explicitly, but this is probably not the PR for it (should be smaller and more surgical if we do)

Reply:
Not sure that I understand the aka.

But this handles cases in which two non-plugin created (API/UI) objects of different types share an ID. This can be tested with the Postman config I sent over and adversaries/objectives. Creating an objective, then creating / updating an adversary with `PUT` will result in (in the created YAML file) an objective with an `adversary_id` property.

Reply:
Ahh, what you explained is exactly what I meant, as the ids are the uuid4's.  So this is for the case where we have multiple objects sharing the same id of differing types. 👍